packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,37 @@
+# TextDecorationControl
+
+**Notice: This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.**
+
+![TextDecorationControl Element in Inspector Control](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/text-decoration-component.png?raw=true)
+
+
+## Usage
+
+```jsx
+import { __experimentalTextDecorationControl } from '@wordpress/block-editor';
```

Comment:
```suggestion
import { __experimentalTextDecorationControl as TextDecorationControl } from '@wordpress/block-editor';
```

The experimental component can be renamed to `TextDecorationControl` and updated below as well. This would  help with the readability of the example and minimise what would need to be updated when the component is stabilized.